source/extensions/filters/http/well_known_names.h

```diff
@@ -50,6 +50,8 @@ class HttpFilterNameValues {
   const std::string GrpcWeb = "envoy.filters.http.grpc_web";
   // GRPC Field Extraction filter
   const std::string GrpcFieldExtraction = "envoy.filters.http.grpc_field_extraction";
+  // Proto Message Logging filter
+  const std::string ProtoMessageLogging = "envoy.filters.http.proto_scrubbing_logging";
```

Comment:
Is this updated?

Reply:
Missed that. Updated it now.

Thanks for noticing!